homeassistant/components/mpd/media_player.py

```diff
@@ -251,6 +252,26 @@ def media_album_name(self):
         """Return the album of current playing media (Music track only)."""
         return self._currentsong.get("album")
 
+    @property
+    def media_image_hash(self):
+        """Hash value for media image."""
+        file = self._currentsong.get("file", None)
+        if file:
+            return hashlib.sha256(file.encode("utf-8")).hexdigest()[:16]
+
+        return None
+
+    async def async_get_media_image(self):
+        """Fetch media image of current playing image."""
+        file = self._currentsong.get("file", None)
+        if file:
+            response = self._client.albumart(file)
+            if response:
+                image = bytes(response)
+                return (image, "image/png")
```

Comment:
I have tested that this also works for other image formats like `image/jpeg`. Not sure if browsers use a quirks mode to do this. If this is not sufficient we'd need to introduce some library that detects mime types, which I think would probably be overkill.